packages/flame/lib/src/widgets/animation_widget.dart

```diff
@@ -48,22 +52,27 @@ class SpriteAnimationWidget extends StatelessWidget {
     required String path,
     required SpriteAnimationData data,
     Images? images,
+    VoidCallback? onComplete,
```

Comment:
```suggestion
    this.onComplete,
```
I don't think onComplete needs to be private, and we can set it directly here.

Reply:
@willhlas, did you see this?

Reply:
@spydon just pushed the update, sorry for the delay!